management-api/src/main/java/com/afh/skytouch/managment/controllers/GenericProductController.java

```diff
@@ -14,20 +16,33 @@
 @RequestMapping("/genericProduct")
 public class GenericProductController {
 
-    @GetMapping("/{uuid}")
-    public ResponseEntity<?> getProduct(@PathVariable UUID uuid){
-        return ResponseEntity.ok().build();
+    private static final String productAtributeName = "product";
```

Comment:
In the same way as suggested in ProductTransferConfiguration, get these constants out of this class and keep them separate. Uppercase them as well.